igor-web/src/main/groovy/com/netflix/spinnaker/igor/travis/service/TravisResultConverter.groovy

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2016 Schibsted ASA.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.igor.travis.service
+
+import com.netflix.spinnaker.igor.build.model.Result
+import groovy.transform.CompileStatic
+import groovy.util.logging.Slf4j
+
+@CompileStatic
+@Slf4j
+class TravisResultConverter {
+    static Result getResultFromTravisState(String state) {
+        switch(state) {
+            case "started":
+                Result.BUILDING
+                break
+            case "passed":
+                Result.SUCCESS
+                break
+            case "failed":
+                Result.FAILURE
+                break
+            case "errored":
+                Result.FAILURE
+                break
+            default:
+                log.info ("could not convert ${state} to result, setting FAILURE")
```

Comment:
It might be better to throw an exception here, since it's pointing at something wrong with the Igor/Travis interface rather than the build failing.